docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -5459,6 +5474,11 @@ image](#create-an-image) section for more details.
                   - **Options** - key/value map of driver specific options
         - **StopGracePeriod** – Amount of time to wait for the container to terminate before
           forcefully killing it.
+        - **DNSConfig** – Specification for DNS related configurations in
```

Comment:
I am not sure if swagger solves the actual problem. Can godoc integrate with swagger?